tests/tensorflow/data/configs/mask_rcnn_coco2017_magnitude_sparsity_int8.json

```diff
@@ -32,11 +32,6 @@
         },
         {
             "algorithm": "quantization",
-            "initializer": {
-                "range": {
-                    "num_init_samples": 1
-                }
-            },
```

Comment:
> BTW, something is wrong in pre-commit with eval of masks. Looks like initialization is configured incorrectly.
> 
> ```
> FAILED tests/tensorflow/test_sanity_sample.py::test_model_eval[segmentation-mask_rcnn_coco2017_magnitude_sparsity_int8.json-coco2017-tfrecords-3]
> ```
> 
> ```
> if range_init_args is None:
> 23:33:34  >           raise ValueError(
> 23:33:34                  'Should run range initialization as specified via config,'
> 23:33:34                  'but the initializing data loader is not provided as an extra struct. '
> 23:33:34                  'Refer to `NNCFConfig.register_extra_structs` and the `QuantizationRangeInitArgs` class')
> 23:33:34  E           ValueError: Should run range initialization as specified via config,but the initializing data loader is not provided as an extra struct. Refer to `NNCFConfig.register_extra_structs` 
> ```

This is a WA for the issue. Created a task 58869 for a proper fix - evaluation with initialization is not valid now, at least in segmentation sample